packages/entities/entities-gateway-services/src/components/GatewayServiceForm.vue

```diff
@@ -430,14 +501,26 @@ const props = defineProps({
     required: false,
     default: false,
   },
+  /** Whether show or hide Try sample API button */
+  hideTrySampleApiButton: {
+    type: Boolean,
+    required: false,
+    default: false,
+  },
+  sampleApiList: {
+    type: Array,
+    required: false,
+    default: ()=> [],
```

Comment:
```suggestion
  sampleApiList: {
```

question: do you need this? I don't think you use it anywhere?

Reply:
removed